### ISSUE TITLE
escape is not batchable

### ISSUE THREAD

**Issue by pascal-hofmann** [thephpleague/plates#96]
escape is not batchable since it has been moved from a default extension to the Template class. So the example:

```
<p>Welcome <?=$this->batch($name, 'strip_tags|strtoupper|escape')?></p>
```

on http://platesphp.com/templates/functions/ does not work. I get a LogicException: 

> The batch function could not find the "escape" function.

Background: I want to do 

```
$this->batch($str, 'escape|nl2br')
```

**Comment by ricardofiorani** [comment]
@phofmann-trust what PHP version are you using ?

**Comment by phofmann-trust** [comment]
I was using PHP 5.5 at that time.

**Comment by ragboyjr** [comment]
@pascal-hofmann have you tried

```
$this->escape($str, 'nl2br|strtoupper');
```

This should batch those two funcs first, then escape the string.

**Comment by ragboyjr** [comment]
@pascal-hofmann does this resolve your issue?

**Comment by reinink** [comment]
This issue was happening because the template methods were not public. This has now been fixed! See 3688ad52a887e15dc1484bf8ffa8b12c7c424061.
